chatterbox/src/Main/MainArea/ChattingComponent/Sidebar/index.css

```diff
@@ -0,0 +1,104 @@
+
+.rectangle-4 
+{
+    height: 939px;
+    width: 411px;
+    background-color: #FFFFFF;
+    box-shadow: 0 2px 12px 3px rgba(0,0,0,0.14);
+    margin-top: 90px;
+    padding-top: 30px;
+    
+}
+.rectangle-7 
+{	
+    height: 43px;	
+    width: 310px;	
+    border: 1px solid #979797;	
+    border-radius: 21.5px; 
+    margin-left: 34px;
+    margin-bottom: 30px;
+}
+.rectangle-8 
+{	
+    height: 139px;	
+    width: 411px;	
+    background-color: rgb(255, 255, 255); 
+    padding-top: 10px; 
+}
+.rectangle-5 
+{	
+    height: 43px;	
+    width: 345px;	
+    border-radius: 21.5px;	
+    background-color: #EAEAEA; 
+    margin-left: 34px; 
+    margin-top: 20px
+}
+.the-practice-of-ciga 

```

Comment:
What is this for?

Reply:
For the text in the chat. We copied and pasted the code from the template without amending the names.